JavaScript/packages/recognizers-text/src/extractors.ts

```diff
@@ -17,6 +17,14 @@ export class ExtractResult {
         return ((er2.start < er1.start) && ((er2.start + er2.length) >= (er1.start + er1.length)))
         || ((er2.start <= er1.start) && ((er2.start + er2.length) > (er1.start + er1.length)));
     }
+	
+	static isCrossOverlap(er1: ExtractResult, er2: ExtractResult): boolean {
```

Comment:
Please always add test cases to show that the problem was fixed. Also try to add other similar cases and cases that may be ambiguous to show that the fix doesn't break other situations, ok?
Thanks.